css/style.css

```diff
@@ -0,0 +1,3958 @@
+
+.accent-header {
+  color: var(--accent-color);
+}
+
+.hero-text {
+  margin-block: 16px 36px;
+  color: var(--opacity-70);
+}
+
+.hero-text::first-letter {
+  text-transform: capitalize;
+}
+
+.btns-header {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-column-gap: 24px;
+  -moz-column-gap: 24px;
+  column-gap: 24px;
+}
+
+.btn-shop {
+  max-width: 170px;
+  text-wrap: wrap;
+  padding: 12px 46px;
+  border-radius: var(--border-radius-s);
+  background-color: var(--accent-color);
+  color: var(--secondary-color);
+  text-transform: capitalize;
+  -webkit-transition: background var(--trasition-effect);
+  -o-transition: background var(--trasition-effect);
+  transition: background var(--trasition-effect);
+}
+
+.btn-shop:hover,
+.btn-shop:active,
+.btn-shop:focus {
+  background-color: var(--state-change-color);
+}
+
+.btn-explore {
+  display: -webkit-inline-box;
+  display: -ms-inline-flexbox;
+  display: inline-flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-column-gap: 12px;
+  -moz-column-gap: 12px;
+  column-gap: 12px;
+  color: var(--primary-color);
+  text-transform: capitalize;
+  border-bottom: 1px solid transparent;
+}
+
+.icon-play {
+  display: -webkit-inline-box;
+  display: -ms-inline-flexbox;
+  display: inline-flex;
+  padding: 16px 14px 16px 18px;
+  background-color: var(--secondary-color);
+  border-radius: 25px;
+  -webkit-box-shadow: var(--box-shadow-s);
+  box-shadow: var(--box-shadow-s);
+}
+
+.icon-play-img {
+  fill: var(--accent-color);
+}
+
+.btn-explore-text {
+  border-bottom: 1px solid transparent;
+  -webkit-transition: border var(--trasition-effect);
+  -o-transition: border var(--trasition-effect);
+  transition: border var(--trasition-effect);
+}
+
+.btn-explore:hover .btn-explore-text,
+.btn-explore:active .btn-explore-text,
+.btn-explore:focus .btn-explore-text {
+  border-bottom: 1px solid var(--primary-color);
+}
+
+.hero-images {
+  position: relative;
+  overflow: hidden;
+}
+
+.hero-images::before {
+  content: "";
+  display: block;
+  padding: 10px;
+  width: 56px;
+  height: 56px;
+  background-image: url("../img/icons/icon-shapes.svg");
+  background-repeat: no-repeat;
+  background-size: 34px;
+  background-position: center;
+  background-color: var(--secondary-color);
+  border-radius: 5px;
+  position: absolute;
+  top: 8px;
+  right: 60px;
+  z-index: 1;
+  -webkit-box-shadow: var(--box-shadow-m);
+  box-shadow: var(--box-shadow-m);
+}
+
+.hero-images::after {
+  content: "";
+  display: block;
+  padding: 10px;
+  width: 56px;
+  height: 56px;
+  background-image: url("../img/icons/icon-palettes.svg");
+  background-repeat: no-repeat;
+  background-size: 32px;
+  background-position: center;
+  background-color: var(--secondary-color);
+  border-radius: 5px;
+  position: absolute;
+  bottom: 40px;
+  left: 30px;
+  z-index: 1;
+  -webkit-box-shadow: var(--box-shadow-m);
+  box-shadow: var(--box-shadow-m);
+}
+
```

Comment:
A lot of the same styles on `::before` and `::after` pseudoelements - use group selector here

Reply:
#Fixed